proposed/2022/pack-referenced-projects.md

```diff
@@ -0,0 +1,64 @@
+# Pack referenced projects
+
+- Author Name (https://github.com/maxkoshevoi)
+- Start Date (2022-03-28)
+- GitHub Issue (https://github.com/NuGet/Home/issues/3891)
+- GitHub PR (-)
+
+## Summary
+
+Allow referenced projects' artifacts to be packed inside the NuGet.
+
+## Motivation
+
+When SDK-style project is being packed as a NuGet, all projects that it references are considered as NuGet themselves that this NuGet depends on.
+This is not always a desired behavior, sometimes user needs artifacts from referenced project(s) to be packed into the NuGet.
```

Comment:
I don’t really understand this. Why wouldn’t another package transitively be adding those resources instead? Packaging the contents could create conflicts when you want to reference a newer version of a transitive reference.
If there are issues with transitively adding resources, why not address this scenario better without packing things into the package?
Could you also add some user stories around these problematic scenarios, beyond just stating it is an issue?